fix: Avalanche exception

When adding an Avalanche endpoint I'm getting this exception within a minute.

```
panic: unexpected excess blob gas value in clique

goroutine 11218 [running]:
github.com/ethereum/go-ethereum/consensus/clique.encodeSigHeader({0x7f2715389620, 0xc000ce0700}, 0xc003c82f00)
	/go/pkg/mod/github.com/ethereum/go-ethereum@v1.13.11/consensus/clique/clique.go:770 +0x605
github.com/ethereum/go-ethereum/consensus/clique.SealHash(0x1?)
	/go/pkg/mod/github.com/ethereum/go-ethereum@v1.13.11/consensus/clique/clique.go:727 +0x7b
github.com/0xPolygon/panoptichain/api.Ecrecover(0xc00140d180?)
	/app/api/validator.go:119 +0x58
github.com/0xPolygon/panoptichain/observer.(*EmptyBlockObserver).Notify(0xc0002eaf50, {0x6c2698?, 0xc000442180?}, {0x1ad5ab8, 0xc001512000})
	/app/observer/rpc.go:34 +0xae
github.com/0xPolygon/panoptichain/observer.(*EventBus).Publish.func1({0x1acf730?, 0xc0002eaf50?})
	/app/observer/observer.go:119 +0x3e
created by github.com/0xPolygon/panoptichain/observer.(*EventBus).Publish in goroutine 232
	/app/observer/observer.go:118 +0x148
```

--- a/api/validator.go
+++ b/api/validator.go
@@ -116,6 +116,13 @@ func Ecrecover(header *types.Header) ([]byte, error) {
 		return nil, errors.New("unable to recover signature")
 	}
 	signature := header.Extra[start:]
+
+	// Set these to nil or clique.SealHash will panic.
+	header.WithdrawalsHash = nil
+	header.BlobGasUsed = nil
+	header.ExcessBlobGas = nil
+	header.ParentBeaconRoot = nil
+
 	pubkey, err := crypto.Ecrecover(clique.SealHash(header).Bytes(), signature)
 	if err != nil {
 		return nil, err

--- a/api/validator.go
+++ b/api/validator.go
@@ -111,18 +111,19 @@ func Signers(n network.Network) (map[string]*Validator, error) {
 
 // Ecrecover recovers the block signer given the block header.
 func Ecrecover(header *types.Header) ([]byte, error) {
+	// These values will cause clique.SealHash to panic.
+	if header.WithdrawalsHash != nil ||
+		header.BlobGasUsed != nil ||
+		header.ExcessBlobGas != nil ||
+		header.ParentBeaconRoot != nil {
+		return nil, errors.New("unable to encode clique header")
+	}
+
 	start := len(header.Extra) - crypto.SignatureLength
 	if start < 0 || start > len(header.Extra) {
 		return nil, errors.New("unable to recover signature")
 	}
 	signature := header.Extra[start:]
-
-	// Set these to nil or clique.SealHash will panic.
-	header.WithdrawalsHash = nil
-	header.BlobGasUsed = nil
-	header.ExcessBlobGas = nil
-	header.ParentBeaconRoot = nil
-
 	pubkey, err := crypto.Ecrecover(clique.SealHash(header).Bytes(), signature)
 	if err != nil {
 		return nil, err

